src/maheshKJun20/NumbersExample/FibonacciNumbers.java

```diff
@@ -0,0 +1,24 @@
+package maheshKJun20.NumbersExample;
+
+public class FibonacciNumbers {
+	
+	void displayFibonacciNumber(int num) {
+		int firstValue = 1, secondValue = 1, newValue = 0;
+		System.out.print("Fibonacci Series for " + num + " Output: 1 1");
```

Comment:
Don't print 1 1 hardcoded. 
Initialize **int firstValue = 1, secondValue = 0, newValue = 0;** and few tweaks in for loop and you'll get expected output

Reply:
Used firstValue and secondvalue variable value in SOP statement. Thanks for suggestion